radarr/movie.go

```diff
@@ -121,6 +121,8 @@ type AddMovieInput struct {
 // AddMovieOptions are the options for finding a new movie.
 type AddMovieOptions struct {
 	SearchForMovie bool `json:"searchForMovie"`
+	// Allowed values: "movieOnly", "movieAndCollection", "none"
+	Monitored string `json:"monitored,omitempty"`
```

Comment:
i think it's `monitor`, not `monitored`, but I checked with v4.2.4.6635 only

Reply:
You are correct (as is the GH issue about it) https://github.com/Notifiarr/notifiarr/issues/370